core/api/src/services/ledger/facade/onchain-send.ts

```diff
@@ -168,7 +169,7 @@ export const setOnChainTxIdByPayoutId = async ({
 
   const bankOwnerWalletId = await getBankOwnerWalletId()
   const bankOwnerTxns = txns.filter(
-    (txn) => txn.satsFee && txn.walletId === bankOwnerWalletId,
+    (txn) => !isOnChainFeeReconciliationTxn(txn) && txn.walletId === bankOwnerWalletId,
```

Comment:
can reconciliation be applied to other wallets (different from bankowner)?

Reply:
I don't see why we would do that right now. `recordReceiveOnChainFeeReconciliation` construct entries that only involve the `Liabilities:bankOwner` and `Assets:OnChain` via the `FeeOnlyEntryBuilder`

Reply:
Mm, it shouldn't be since reconciliations only involve `Liabilities:bankOwner` and `Assets:OnChain`. Why do you ask?

Reply:
if that is the case then walletId validation should be in `isOnChainFeeReconciliationTxn`

Reply:
Not sure if that would be helpful here though, because we'd still need to do the `txn.walletId === bankOwnerWalletId` check since normal non-reconciliation `onchain_payment` txns can also be from user wallet ids